Lib/test/test_descr.py

```diff
@@ -4111,18 +4111,6 @@ def __rfloordiv__(self, other):
         self.assertEqual(D() // C(), "D.__floordiv__")
         self.assertEqual(C() // D(), "D.__rfloordiv__")
 
-        # Case 4: this didn't work right in 2.2.2 and 2.3a1
-
-        class E(C):
-            pass
-
-        self.assertEqual(E.__rfloordiv__, C.__rfloordiv__)
-
-        self.assertEqual(E() // 1, "C.__floordiv__")
-        self.assertEqual(1 // E(), "C.__rfloordiv__")
-        self.assertEqual(E() // C(), "C.__floordiv__")
-        self.assertEqual(C() // E(), "C.__floordiv__") # This one would fail
```

Comment:
Why are these being taken out?  I don't see div or floordiv in the above, so it doesn't look like it is just being tested differently (unless the tests were really redundant or something).

Reply:
The tests here seemed redudant with the tests I added for add above, since the logic is not operator specific. I removed these specific testa because they were verifying the old (changed) behavior, but I could also keep them in and change to the new behavior.